src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,29 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Numbers {
+
+    private final List<Number> numbers;
+
+    public Numbers(List<String> textNumbers) {
+        this.numbers = rapping(textNumbers);
+    }
+
+    private List<Number> rapping(List<String> stringNumbers) {
+        List<Number> numbers = new ArrayList<>();
+        for (String textNumber : stringNumbers) {
+            numbers.add(new Number(textNumber));
+        }
+        return numbers;
```

Comment:
```suggestion
        return stringNumbers.stream()
                .map(Number::new)
                .collect(Collectors.toList());
```
이렇게 간결하게 표현할 수도 있습니다 :)